.gitignore

```diff
@@ -1,10 +1,9 @@
-.idea/*
-venv/*
-source/.idea/*
-source/__pycache__/*
-out/tensorflow_pipeline/logs/*
+__pycache__/
+.idea/
+venv/
+out/tensorflow_pipeline/logs/
```

Comment:
**/logs/

Reply:
Just to clarify that without wildcards I think github is assuming .idea/ and venv/ are in the root folder

Reply:
Same here as above. I will change to `logs/`. I did some reading and I think what you're referring to is `/logs/` which only matches the root directory folder.